### DISCUSSION TITLE
Node.js 18.x LTS works fine on Win7 x64 ESU

### DISCUSSION THREAD

**Discussion by spacedrone404** [Alex313031/codium#6] [Ideas]
just for a general information

**Comment by Alex313031** [comment]
@spacedrone404 I know! I made patched builds of Node 14, 16, and 18 for Windows 7. I use them along with Visual Studio 2019 to build the Codium releases.

Specifically, I usually use Node 16 > https://github.com/Alex313031/node16-win7

However, this still doesn't mean that I can use a newer version of Electron that uses Node 18.
